### ISSUE TITLE
maven build failure

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#3269]
https://travis-ci.org/ElektraInitiative/libelektra/jobs/615962665

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.12.4:test (default-test) on project libelektra4j: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.12.4:test failed: Plugin org.apache.maven.plugins:maven-surefire-plugin:2.12.4 or one of its dependencies could not be resolved: Could not transfer artifact org.apache.commons:commons-lang3:jar:3.1 from/to google-maven-central (https://maven-central.storage-download.googleapis.com/maven2/): GET request of: org/apache/commons/commons-lang3/3.1/commons-lang3-3.1.jar from google-maven-central failed: Connection reset -> [Help 1]

[ERROR] 

[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.

[ERROR] Re-run Maven using the -X switch to enable full debug logging.

[ERROR] 

[ERROR] For more information about the errors and possible solutions, please read the following articles:

[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
```

**Comment by Piankero** [comment]
Did this occur in other places too?

**Comment by markus2330** [comment]
I do not know.

Is this maybe only some network problem?

What about adding `-e` to the mvn calls?

**Comment by Piankero** [comment]
> Is this maybe only some network problem?

Thats why I was asking. To me it looks like a temporary network issue.

> What about adding -e to the mvn calls?

The main message is there so I do not think that we need an extra stacktrace

Let's wait some time and see if this happens again.

**Comment by markus2330** [comment]
> The main message is there so I do not think that we need an extra stacktrace

Yes, but in general `-e` might be better? If it is successful, we will not see anything anyway?

**Comment by Piankero** [comment]
> Yes, but in general -e might be better? If it is successful, we will not see anything anyway?

In your posted message the relevant part is already given (`Connection reset`). Adding `-e` to it would just give you an additional huge stacktrace that is unnecessary.

**Comment by markus2330** [comment]
Lets better turn to more important things.

**Comment by markus2330** [comment]
Happened again: https://github.com/ElektraInitiative/libelektra/pull/3549/checks?check_run_id=1492810253

**Comment by mpranj** [comment]
Starting to occur more often now, especially on Fedora/Cirrus-CI: https://cirrus-ci.com/task/4827231149096960

**Comment by markus2330** [comment]
Ok, then let us reopen this. Maybe we fix it by switching to Gradle as proposed by @tucek.

**Comment by tucek** [comment]
to be closed when PR #3714 is merged.
